runtime/api.proto

```diff
@@ -0,0 +1,1227 @@
+// To regenerate api.pb.go run hack/update-generated-runtime.sh
+syntax = 'proto3';
+
+package runtime.v1alpha2;
+option go_package = "v1alpha2";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) =  true;
+option (gogoproto.goproto_getters_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unrecognized_all) = false;
+
+// Runtime service defines the public APIs for remote container runtimes
```

Comment:
Should the path to this file include something something about CRI or Kubernetes?

Reply:
You mean to change the `runtime` directory name?

Reply:
yeah! Otherwise it looks like this is the scope runtime module or something.